### PR TITLE
Making bundle viz links collapsable

### PR DESCRIPTION
**Pull request by ryanml**
Also contains some code cleanup of bundle name parsing

### CODE REVIEW

File: development/metamaskbot-build-announce.js [hunk 1]
```diff
@@ -92,7 +85,10 @@ async function start() {
 
   const contentRows = [
     `builds: ${buildLinks}`,
-    `bundle viz: ${bundleMarkup}`,
+    `<details>
```
Comment (ryanml) [inline development/metamaskbot-build-announce.js:88]: I would have liked to clean up the spacing some by applying `margin-bottom: 0` here, but looks like GH strips attributes

Comment (ryanml) [reply development/metamaskbot-build-announce.js]: I moved the bundle viz links to the last row and this solves the aforementioned issue

**Review by rekmarks — APPROVED** [review]
LGTM! Nice regex 😎

**Review by NiranjanaBinoy — APPROVED** [review]
LGTM

**Pull request merged**